assets/sumologic.yaml

```
@@ -22,21 +23,28 @@ extensions:
       on_rebound: true
     directory: /var/lib/otelcol-sumo/file_storage
 
+  ## Configuration for OpAMP Extension
+  ## The OpAMP extension remotely manages collector configuration.
+  ## ref: https://github.com/SumoLogic/sumologic-otel-collector/tree/main/pkg/extension/opampextension
+  opamp:
+    remote_configuration_directory: /etc/otelcol-sumo/opamp.d
+    endpoint: wss://opamp-events.sumologic.com/v1/opamp
+
 receivers:
   ## Configuration for OTLP Receiver
   ## Receives data via gRPC or HTTP using OTLP format.
   ## ref: https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/otlpreceiver
   otlp:
     protocols:
       grpc:
-        endpoint: 0.0.0.0:4317
+        endpoint: localhost:4317
       http:
-        endpoint: 0.0.0.0:4318
+        endpoint: localhost:4318
 
 exporters:
   ## Configuration for Sumo Logic Exporter
   ## This exporter supports sending logs, metrics and traces data to Sumo Logic.
-  ## ref: https://github.com/SumoLogic/sumologic-otel-collector/tree/main/pkg/exporter/sumologicexporter
+  ## ref: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.102.0/exporter/sumologicexporter
```

```suggestion
  ## ref: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/sumologicexporter
```

Let's go with main as we do not update the version in this repo AFAIK

---

It seems like we do create tags for each release in the contrib repository. I'd prefer to link to a tag, even if it's old, as we can guarantee that the URL will work. If we link to the main branch and we ever choose to move files around, the URL may be invalid.

---

We should ensure that we update the link then, because linking to configuration which doesn't work is not so good and it may be harder to spot it than `404`

---

That could happen if we link to tip too, it seems to me.

---

That is, if you had an older version, configuration for it may be incompatible with whatever's in tip.

---

Yeah, my thinking is that we are most likely use the latest version than update this link if we do not automate it 😓